deep_mlp.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/python
```

Comment:
Since this file is tied to the `DigitRecognition` example, maybe we can move it inside that folder?

Reply:
Yes.